packages/contracts/contracts/libs/v0.8.x/minter-libs/AuthLib.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+import "../../../interfaces/v0.8.x/IGenArt721CoreContractV3_Base.sol";
+import "../../../interfaces/v0.8.x/IMinterFilterV1.sol";
+
+/**
+ * @title Art Blocks Authorization Minter Library
+ * @notice This library contains helper functions that may be used by minters
+ * to check authorization for performing operations.
+ * @author Art Blocks Inc.
+ */
+
+library AuthLib {
```

Comment:
On a related note, I'm inclined that this library lives in the higher level libs directory rather than minter specific, because I can see us using these auth lib methods for other things down the line.

Reply:
yeah - sorry this could be its own PR, but would prefer to just add to other minters in follow-on PR at this point (ABHelpers as well). good point that this could have been done as a separate PR to achieve a smoler PR 😅 
re: directory - good call, and going to bump this up a level (to be consistent with ABHelpers.sol)

Reply:
updated in [ac464f5](https://github.com/ArtBlocks/artblocks-contracts/pull/871/commits/ac464f55e43c194a9c026ed099b88bf418c0ed23)

Reply:
SGTM ser – thank you! 🙏 💜